sphinx_gallery/notebook.py

```diff
@@ -142,36 +145,70 @@ def add_code_cell(work_notebook, code):
     work_notebook["cells"].append(code_cell)
 
 
-def add_markdown_cell(work_notebook, text):
+def add_markdown_cell(work_notebook, text, pandoc=False):
     """Add a markdown cell to the notebook
 
     Parameters
     ----------
     code : str
         Cell content
     """
-    markdown_cell = {
-        "cell_type": "markdown",
-        "metadata": {},
-        "source": [rst2md(text)]
-    }
+    if pandoc:
+        markdown_cell = {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [text]
+        }
+    else:
+        markdown_cell = {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [rst2md(text)]
+        }
```

Comment:
```suggestion
    if not pandoc:
        text = rst2md(text)
    markdown_cell = {
        "cell_type": "markdown",
        "metadata": {},
        "source": [text]
    }
```

Reply:
And actually `pandoc` as a kwarg is not so informative here. Maybe `convert=True` should be the default, meaning "convert RST to MD" and in whatever code calls this sets `convert=not pandoc`